src/cares_wrap.cc

```diff
@@ -297,6 +297,64 @@ Local<Array> HostentToNames(Environment* env, struct hostent* host) {
 }
 
 
+/**
+ * This function is to check whether current servers are fallback servers
+ * when cares initialized.
+ *
+ * The fallback servers of cares is [ "127.0.0.1" ] with no user additional
+ * setting.
+ */
+void AresCheckServers(Environment* env) {
```

Comment:
tiny nit: `Check` sounds a bit like it’s just testing things, maybe `Ensure` is better?